roles/project/templates/job_definition.yml.j2

```diff
@@ -30,10 +30,27 @@ spec:
               value: "{{ project_organization }}"
             - name: PROJECT_DESCRIPTION
               value: "{{ project_description }}"
```

Comment:
Now that description fields are not required, we probably need to similarly conditionally template this.